argoverse/utils/interpolate.py

```diff
@@ -155,26 +155,31 @@ def eliminate_duplicates_2d(px: np.ndarray, py: np.ndarray) -> Tuple[np.ndarray,
     return px, py
 
 
-def interp_arc(t: int, px: np.ndarray, py: np.ndarray) -> np.ndarray:
-    """Linearly interpolate equally-spaced points along a polyline.
+def interp_arc(t: int, px: np.ndarray, py: np.ndarray, pz: np.ndarray = None) -> np.ndarray:
```

Comment:
`pz` should be made type `Optional[np.ndarray]` or initialized to something other than `None`.